.github/actions/quic-interop-runner/action.yml

```diff
@@ -92,9 +92,8 @@ runs:
       run: |
         echo '[**QUIC Interop Runner**](https://github.com/quic-interop/quic-interop-runner)' >> comment
         echo '' >> comment
-        echo '```' >> comment
-        cat quic-interop-runner/summary >> comment
-        echo '```' >> comment
+        # Ignore all, but table, which starts with "|:--".
+        cat quic-interop-runner/summary | awk '/^\|:--/{flag=1} flag' >> comment
```

Comment:
Why not just `grep '^\|:--' quic-interop-runner/summary >> comment` ?

Reply:
:+1: created a follow-up @martinthomson https://github.com/mozilla/neqo/pull/1747.